source/welcome/what-changed-in-v70.rst

```diff
@@ -0,0 +1,108 @@
+What changed in Mattermost v7.0?
+================================
+
+|all-plans| |cloud| |self-hosted|
+
+.. |all-plans| image:: ../images/all-plans-badge.png
+  :scale: 30
+  :target: https://mattermost.com/pricing
+  :alt: Available in Mattermost Free and Starter subscription plans.
+
+.. |enterprise| image:: ../images/enterprise-badge.png
+  :scale: 30
+  :target: https://mattermost.com/pricing
+  :alt: Available in the Mattermost Enterprise subscription plan.
+
+.. |professional| image:: ../images/professional-badge.png
+  :scale: 30
+  :target: https://mattermost.com/pricing
+  :alt: Available in the Mattermost Professional subscription plan.
+
+.. |cloud| image:: ../images/cloud-badge.png
+  :scale: 30
+  :target: https://mattermost.com/download
+  :alt: Available for Mattermost Cloud deployments.
+
+.. |self-hosted| image:: ../images/self-hosted-badge.png
+  :scale: 30
+  :target: https://mattermost.com/deploy
+  :alt: Available for Mattermost Self-Hosted deployments.
+
+.. |view-info-icon| image:: ../images/information-outline_F02FD.svg
+  :height: 24px
+  :width: 24px
+  :alt: Access channel information in the right pane using the View Info icon.
+
+.. |message-actions-icon| image:: ../images/apps_F003B.svg
+  :height: 24px
+  :width: 24px
+  :alt: Access message actions to push information to an external system through a Mattermost integration using the Message actions icon.
+
+.. |copy-link-icon| image:: ../images/paperclip_F03E2.svg
+  :height: 24px
+  :width: 24px
+  :alt: Copy the public URL link for an image in a message using the Copy link icon.
+
+.. |more-actions-icon| image:: ../images/dots-horizontal_F01D8.svg
+  :height: 24px
+  :width: 24px
+  :alt: Access additional message actions using the More actions icon.
+
+
+In Mattermost v7.0, we've changed where you can find channel information and where you can access your Mattermost integrations and their related actions. Whether you're new to Mattermost, or a long-time user, this document will help you start using these new features and introduce you to new ways of navigating the product.
+
+Channel information
+-------------------
+
+When creating channels, you're prompted to provide context that helps other users understand the focus of the channel. From Mattermost v7.0, these channel details appear in both the center pane under the channel name and in a new Channel information screen. 
+
+You can show or hide channel information in the right pane by selecting the **View info** |view-info-icon| icon located in the top right corner of the center pane, or by using the keyboard shortcut CMD/CTRL+ALT+I.
+
+.. image:: ../images/channel-info.png
+    :alt: Access channel information by using the View info icon.
+
+.. tip::
+
+    You can continue managing both the channel's purpose and header for any channel created prior to upgrading to Mattermost v7.0. New channels created using Mattermost v7.0 or later include only a channel header you can manage.
```

Comment:
I think both purpose and header are still available in v7.0? At least they should be. It's just that the create channel modal doesn't ask for the header to be added at the time of creation (no docs needed here though)